.github/Dockerfile.c7-network-role

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/centos/centos:centos7
+
+RUN yum -y install epel-release && \
+    yum -y upgrade && \
+    yum -y install NetworkManager NetworkManager-wifi \
+    procps-ng iproute ansible openssh-server openssh-clients \
+    dnsmasq hostapd wpa_supplicant openssl ethtool iputils
```

Comment:
AFAIK, it is also recommended to run "yum clean all" at the end to avoid problems with outdated yum caches in case any of the tests install something, too.

Reply:
Added. Thanks!